e2e_test/error_ui/main.slt

```diff
@@ -15,8 +15,9 @@ alter system set not_exist_key to value;
 ----
 db error: ERROR: QueryError
 
-Caused by this error:
-  1: internal error: SystemParams error: unrecognized system param "not_exist_key"
+Caused by these errors (recent errors listed first):
+  1: gRPC request failed: Internal error
+  2: SystemParams error: unrecognized system param "not_exist_key"
```

Comment:
Do you have any other more interesting example to demonstrate the change? 🤪 I feel this change isn't "aha" enough.

Reply:
> Do you have any other more interesting example to demonstrate the change? 🤪 I feel this change isn't "aha" enough.

I guess more examples will be in the developer logs (like actor bugs or network failure, not reviewed or refactored for now), as the frontend has cached the major stuff and done the check itself.

Reply:
> Besides, I'm wondering whether it's possible to have sth like `gRPC request to xx node failed` 🤪

Having the attempt. Stay tuned. 😄